pep-0590.rst

```diff
@@ -308,7 +308,7 @@ References
 .. [2] tp_call/PyObject_Call calling convention
    https://docs.python.org/3/c-api/typeobj.html#c.PyTypeObject.tp_call
 .. [3] Using PY_VECTORCALL_ARGUMENTS_OFFSET in callee
-   https://github.com/markshannon/cpython/blob/vectorcall-minimal/Objects/classobject.c#L53
+   https://github.com/markshannon/cpython/blob/815cc1a30d85cdf2e3d77d21224db7055a1f07cb/Objects/classobject.c#L53
```

Comment:
Did you check this (and the others) are actually still pointing to the correct line? :)

Reply:
Yup, I checked each one carefully; since they were from pretty recent PEPs, they all fortunately were.